tests/testsuite.py

```diff
@@ -88,12 +88,12 @@ def __init__(self, testname, timeout=10, env=Ellipsis, log=sys.stdout,
     if testname[0] != '/':
       testname = './' + testname
 
-    if script == True:
-        # Run the wrapper script with dash.
-        testname = '/usr/bin/dash ' + testname
+    testname = '/usr/bin/dash -c ' + '\''+ testname + '\''
```

Comment:
Not many systems have dash installed (mine don't). Normally one should use the system shell for this, not a specfic
one (i.e. /usr/bin/sh). IIRC, dash was used because bash triggered leak complaints in valgrind, causing these few,
and only those, to fail on my system (for instance).  But this change makes all tests fail for me.

So, I think this needs to be done different, either by returning to using sh(1) (and somehow suppressing the valgrind fails), or by fixing whatever problem valgrind has with calling exes with libpulp directly, or by at least
using the unconditional wrapping via shell only when using valgrind (i.e. when $TESTS_THROUGH_VALGRIND is
set).

Reply:
Maybe returning to sh is okay now, as Liska updated the CI to run openSUSE :)